app/views/AppealCoverBothPenaltiesPage.scala.html

```diff
@@ -0,0 +1,37 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.AppConfig
+
+@this(layout: Layout,
+      pageHeader: components.pageHeader,
+      link: components.link
+)
+
+@(nextPage: String)(implicit messages: Messages, appConfig: AppConfig, userRequest: UserRequest[_])
+
+@layout(pageTitle = messages("appealCoverBothPenalties.headingAndTitle"),
+    isPageFullWidth = false,
+    backLinkEnabled = false) {
```

Comment:
The back link should be enabled.
I know routing is going to be done in a separate PR, but that should be related to routing to and from these pages, any interaction within these pages should be dealt with now.
An example can be found in: #337